node/resources/routes.ts

```diff
@@ -0,0 +1,14 @@
+import {HttpClient, InstanceOptions, IOContext} from '@vtex/api'
+import { UserRouteInfo } from 'v1typings'
+
+export default class Routes {
```

Comment:
why not creating a dataSource ? DataSources are currently the "go to" solution. you can learn how it's done in here:
https://github.com/vtex/node-vtex-api/blob/4a025a05140bb6d48b7a1e0b5c05267fbdd0eebe/src/Apps.ts#L49